From: chasey-dev <ellenyoung0912@gmail.com>
Subject: [PATCH] warp: Include vmalloc declarations

BM source files use vmalloc helpers without including their declaring
header. Include linux/vmalloc.h instead of relying on indirect includes.

Signed-off-by: chasey-dev <ellenyoung0912@gmail.com>
--- a/warp_rx_bm.c
+++ b/warp_rx_bm.c
@@ -18,6 +18,7 @@
 #include "wdma.h"
 #include <linux/version.h>
 #include <linux/pci.h>
+#include <linux/vmalloc.h>
 
 static void
 wed_rx_bm_dma_cb_exit(struct wed_entry *wed, struct warp_dma_cb *dma_cb)
--- a/warp_rx_page_bm.c
+++ b/warp_rx_page_bm.c
@@ -19,6 +19,7 @@
 #include "warp_utility.h"
 #include <linux/version.h>
 #include <linux/pci.h>
+#include <linux/vmalloc.h>
 
 static u32
 wed_msdu_page_hash(dma_addr_t pa)
--- a/warp_tx_bm_v2.c
+++ b/warp_tx_bm_v2.c
@@ -18,6 +18,7 @@
 #include "wdma.h"
 #include <linux/version.h>
 #include <linux/pci.h>
+#include <linux/vmalloc.h>
 
 void
 dump_pkt_info(struct wed_buf_res *res)
